spyder/widgets/pathmanager.py

```diff
@@ -202,20 +215,52 @@ def setup(self):
         self.refresh()
 
     @Slot()
-    def synchronize(self):
+    def import_pythonpath(self):
+        """Import from PYTHONPATH environment variable"""
+        env_pypath = os.environ.get('PYTHONPATH', '')
+        if not env_pypath:
+            return

```

Comment:
This is confusing because the button does nothing when clicked, which users could interpret as it being broken. So, in this case I suggest to add a `QMessageBox.information` saying that `PYTHONPATH` is empty, so there's nothing to import from it.

Reply:
No problem, I'll add another modal window.